meteor/server/migration/1_46_1.ts

```diff
@@ -0,0 +1,46 @@
+import { addMigrationSteps } from './databaseMigration'
```

Comment:
Is there any good reason to make this file now? Won't it have to be removed in a merge with NRK? Since they might already be past release 46 on some installations, these migrations might not run when upgrading from some newer releases to whatever release this will be merged into.

Reply:
They have been moved back